config/config_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"path"
 	"testing"
 
+	"github.com/aoscloud/aos_common/aoserrors"
```

Comment:
Add empty line between local and extern packages for all files in this commit.

Reply:
done